src/components/PostCard/PostCard.js

```diff
@@ -26,7 +27,8 @@ const PostCard = ({ post, options = {} }) => {
   }
 
   return (
-    <div className={styles.postCard}>
+    <div className={isSticky ? `${styles.postCard} ${styles.postCardSticky}` : `${styles.postCard}`}>
```

Comment:
Another thing we can consider doing is pulling this up into a variable, might make it easier to read, but i don't feel strongly about it

for instance, could do something like:

```
let className = styles.postCard;

if ( isSticky ) {
  className = `${className} ${styles.postCardSticky}`
}
```

Reply:
Nice! I agree, will make the changes 👍